Projects/UOContent/Misc/AccountPrompt.cs

```diff
@@ -10,9 +10,10 @@ public static void Initialize()
             if (Accounts.Count == 0)
             {
                 Console.WriteLine("This server has no accounts.");
-                Console.Write("Do you want to create the owner account now? (y/n)");
+                Console.Write("Do you want to create the owner account now? (y/n): ");
 
-                if (Console.ReadKey(true).Key == ConsoleKey.Y)
+                var answer = Console.ReadLine();
```

Comment:
Someone will have to press enter right? I wonder if we can avoid that?

Reply:
Yes. Just like when entering admin username/password. I thought its a lesser evil.

Reply:
Now that I have thought about it, I think it is fine.